frontend/src/app/projects/create-project/create-project.component.html

```diff
@@ -3,14 +3,14 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-<div class="wrapper">
-  <mat-horizontal-stepper linear #stepper>
-    <mat-step completed="false" label="General information">
-      <div class="flex-center">
-        <mat-card class="section-card">
+<div class="wrapper flex justify-center">
+  <mat-horizontal-stepper class="w-[710px]" linear #stepper>
+    <mat-step completed="false" label="Create new project">
+      <div class="flex justify-center">
```

Comment:
The background color seems to be slightly different between global background and the div: 
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/703dd01b-e3a2-491e-adf0-b467bec49a0c)

Let's have a quick chat about this after the Daily. I'd like to hear your opinion if we should use the same background color.

Reply:
Added shadow and rounded corners to stepper:
<img width="736" alt="ShadedRoundedStepper" src="https://github.com/DSD-DBS/capella-collab-manager/assets/79602065/19322e25-a818-46f4-bd12-2902ee7a9c28">